frontend/src/redux/types/locationList.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-unused-vars */
+export interface locationsListStateType {
+  locations: any[];
+  bounds: boundsType;
+  zoomPosition: latlngType;
+}
```

Comment:
'boundsType' was used before it was defined.eslint[no-use-before-define](https://eslint.org/docs/rules/no-use-before-define)

Reply:
for latlngType as well

Reply:
And we have had it in types.ts. Why did you comment it? 
![image](https://user-images.githubusercontent.com/41287687/166889767-dc21a858-f90f-4c08-aae4-e521e02553a1.png)